README.md

```diff
@@ -75,34 +79,34 @@ is set, the default is `snowflake`, which uses standard username/password suppor
 
 ```bash
 # no SNOWFLAKE_AUTHENTICATOR needed
-export SNOWFLAKE_PASSWORD="<your-snowflake-password>"
+SNOWFLAKE_PASSWORD="<your-snowflake-password>"
```

Comment:
Why did we remove export?  These env vars won't be visible to the model generator without export.

Reply:
Ah, I wanted to match the example syntax to the generic `.env` format and not have OS specific keywords like `export` or `set`. It's a bit unclear though, I'll add example specific for each OS!